src/panels/mailbox/ha-panel-mailbox.js

```diff
@@ -251,15 +194,9 @@ class HaPanelMailbox extends LocalizeMixin(PolymerElement) {
       });
     }.bind(this));
     return Promise.all(items).then(function (platformItems) {
-      var arrayLength = items.length;
-      var final = [];
-      for (var i = 0; i < arrayLength; i++) {
-        final = final.concat(platformItems[i]);
-      }
-      final.sort(function (a, b) {
+      return [].concat.apply([], platformItems).sort(function (a, b) {
```

Comment:
Ok, one last thing and I think I'm done 😉 : I think the spread operator will be a little cleaner here `[].concat(...platformItems).sort`

Reply:
That is something I haven't seen before.  Cool